mtags/src/main/scala/scala/meta/internal/mtags/CommonMtagsEnrichments.scala

```diff
@@ -134,6 +134,14 @@ trait CommonMtagsEnrichments {
           CompilerRangeParams(params.uri, params.text, start, end, params.token)
       }
     }
+
+    def toOffset: OffsetParams =
```

Comment:
This isn't needed. `RangeParams` has the same methods available and inherit from RangeParams